etc/desimodel_sync_kpno_cron.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+
+# Path to the sync directory, containing the desimodel data trunk
+# checkout and logs.
+syncdir=/n/home/datasystems/desimodel_sync
+logdir="${syncdir}/logs"
+svntrunk="${syncdir}/desimodel_trunk"
+
+logdate="$(date -u --iso-8601=seconds)"
+logname="sync_${logdate}"
+logfile="${logdir}/${logname}"
+
+echo "Running at ${logdate}" > "${logfile}"
+
+# Get the latest desiconda install
+desiconda=$(ls -d /software/datasystems/desiconda/20* | sort | tail -n 1)
+
+# Get the latest KPNO version of desimodules
+desimodules=$(ls ${desiconda}/modulefiles/desimodules/*-kpno | sort | tail -n 1 | xargs basename)
```

Comment:
New comment: 21.3-kpno has been superceded by 21.5 (now the same for both NERSC and KPNO), so this match needs to be updated).

However: simply piping through "sort" picks up 21.9 before 21.10, and ditto for "sort -n".  I haven't found the sorting recipe that works for this case.

Reply:
does "sort -V" help?

Reply:
indeed!  Thanks.  -V / --version-sort is just what I was looking for (and previously unaware of...)